extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -46,7 +46,7 @@ export class AmpIframe extends AMP.BaseElement {
         this.element);
     const containerUrl = parseUrl(containerSrc);
     assert(
-        !((' ' + sandbox + ' ').match(/\s+allow-same-origin\s+/)) ||
+        !((' ' + this.sandbox_ + ' ').match(/\s+allow-same-origin\s+/)) ||
```

Comment:
Ah, I didn't see the `#lowerCase`. Still, the `' ' + this.sandbox_ + ' '` bothers me.